src/content/config.ts

```diff
@@ -49,6 +49,19 @@ export const collections = {
 				.describe(
 					'Summary of this blog post. Appears on the blog index as well as in metadata displayed on social media.',
 				),
+			homepageLink: z
+				.object({
+					title: z.string().max(16).describe('Very short call-out, e.g. `Astro 4.14` or `New!`.'),
```

Comment:
Just checking that whatever we'll have planned re: `Astro 5.0 beta` will fit in 16 characters or less! :smile:

Reply:
That’s 14 characters so… yes? 😄

Reply:
I counted it out lol! (The point is, I checked!)  :stuck_out_tongue: